### ISSUE TITLE
Release v3.2.9

### ISSUE THREAD

**Issue by sauloperez** [openfoodfoundation/openfoodnetwork#6087]
Steps: 

- [x] Include translations: `tx pull --force`
- [x] [Draft new release]
- [x] Notify #instance-managers of user-facing changes.
- [x] Test: https://semaphoreci.com/openfoodfoundation/openfoodnetwork-2/branches/master/builds/1739
- [x] Update translations if necessary
- [x] Publish and notify #global-community
- [x] Deploy and notify #instance-managers
- [x] Nudge next release manager

The full process is described at https://github.com/openfoodfoundation/openfoodnetwork/wiki/Releasing.

[Draft new release]: https://github.com/openfoodfoundation/openfoodnetwork/releases/new?tag=v&title=v+Code+Name&body=Congrats%0A%0ADescription%0A%0A%23%23+User+facing+changes+:eyes:%0A%0A%0A%0A%23%23+Technical+changes+:wrench:%0A%0A

**Comment by filipefurtad0** [comment]
All good.
https://docs.google.com/document/d/1NHpTvwfXWigpV0a-OWFgnUBJwEAuIQFO8R0foq7zkMY/edit#

I checked some pending issues:

- setting-up a Stripe account (in the context of PR #5988 ) - OK
- double checked issue 2) as mentioned [here](https://github.com/openfoodfoundation/openfoodnetwork/issues/5976#issuecomment-697309199) - hub's address does not get into the shipping address fields if the shipping method is "delivery"; it does happen for collection shipping methods - OK

I did verify an issue first observed [here](https://github.com/openfoodfoundation/openfoodnetwork/issues/6043#issuecomment-697317789). This should be in production as well. Opened issue #6090 - I don't think it's a release blocker.
